.sauce/playwright.yml

```diff
@@ -19,6 +19,7 @@ suites:
   - name: "saucy test"
     platformName: "Windows 10"
     testMatch: '**/*.js'
+    screenResolution: "2560x1600"  # Available resolutions on Windows: '800x600', '1024x768', '1152x864', '1280x768', '1280x800', '1280x960', '1280x1024', '1400x1050', '1440x900', '1600x1200', '1680x1050', '1920x1080', '1920x1200', '2560x1600'
```

Comment:
Any reason we want the biggest resolution to be our default?

Reply:
I'll update to `1024x768` 😂

Reply:
I don't think `1024x768` is a reasonable resolution either (unless we are talking about computers more than a decade ago) ;) 
What's the default in our cloud and why not leave it at that?

Reply:
I checked in chef code. If I didn't set any resolution, it would use `1024x768`.